### PR TITLE
merge geo spatial into 2.8

### LINKED ISSUE #1445: Geospaital functionality merge into 2.8
Functionality is done according to #649 requirements

### PR DESCRIPTION
**Pull request by ssuvorov-fls**
fixes #1445

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,17 @@ security.saml.keyManager.defaultKey=apollo - keystore alias
 security.saml.sloUrl=https://localhost:8443/cas/logout - identity provider logout URL
 security.saml.callbackUrl=http://localhost:8080/WebAPI/user/saml/callback - URL called from identity provider after login
 
+## Geospacial support
```
Comment (konstjar) [inline README.md]: Geospa**t**ial typo in title and below

File: README.md [hunk 2]
```diff
@@ -30,6 +30,17 @@ security.saml.keyManager.defaultKey=apollo - keystore alias
 security.saml.sloUrl=https://localhost:8443/cas/logout - identity provider logout URL
 security.saml.callbackUrl=http://localhost:8080/WebAPI/user/saml/callback - URL called from identity provider after login
 
+## Geospatial support
```
Comment (anthonysena) [inline README.md:33]: @wivern - can we move this portion of the README to the https://github.com/OHDSI/webapi-component-geospatial repo instead?

File: pom.xml [hunk 3]
```diff
@@ -220,6 +220,8 @@
     <buildinfo.webapi.milestone.id>25</buildinfo.webapi.milestone.id>
     <buildinfo.atlas.release.tag>*</buildinfo.atlas.release.tag>
     <buildinfo.webapi.release.tag>*</buildinfo.webapi.release.tag>
+
+    <gis.enabled>false</gis.enabled>
```
Comment (anthonysena) [inline pom.xml:224]: Can we add this setting to the `configuration` /info endpoint?

File: README.md (hunk 2 above)
Comment (anthonysena) [reply README.md:33]: #1336 - let's add a page to the wiki to list out optional plugins such as geospatial.

File: pom.xml (hunk 3 above)
Comment (ssuvorov-fls) [reply pom.xml:227]: done

File: README.md (hunk 2 above)
Comment (ssuvorov-fls) [reply README.md:33]: information from README was moved to https://github.com/OHDSI/webapi-component-geospatial

Comment (ssuvorov-fls) [reply README.md:33]: @anthonysena what wiki do you mean - webapi(https://github.com/OHDSI/WebAPI/wiki) or atlas(https://github.com/OHDSI/Atlas/wiki)?

Comment (anthonysena) [reply README.md:33]: Thanks @ssuvorov-fls - appreciate you moving the information to the README. As for the wiki, you can ignore that comment. I was just linking this PR to the issue I'm using to track what items needs to be added to the Atlas/WebAPI wiki. I'll figure out where these types of plugins should be listed so do not worry about those details for this PR.

**Comment by chrisknoll** [comment]
This migration fails on sql server:

```
Message    : The multi-part identifier "ohdsi.sec_permission_id_seq.nextval" could not be bound.
Location   : db/migration/sqlserver/V2.8.0.20200413150815__gis_service_api.sql (C:\Git\WebAPI\target\classes\db\migration\sqlserver\V2.8.0.20200413150815__gis_service_api.sql)
```

The proper syntax for accessing the next value of a sequence is:
NEXT VALUE FOR ${ohdsiSchema}.{sequence name}

This is the problem line:
```
INSERT INTO ${ohdsiSchema}.sec_permission(id, value, description) VALUES
  (${ohdsiSchema}.sec_permission_id_seq.nextval, 'gis:cohort:*:bounds:*:get', 'Get gis bounds for source');
```

Which should be: 
```
INSERT INTO ${ohdsiSchema}.sec_permission(id, value, description) VALUES
  (NEXT VALUE FOR ${ohdsiSchema}.sec_permission_id_seq, 'gis:cohort:*:bounds:*:get', 'Get gis bounds for source');
```

This may be a case where this is a syntax supported in later versions of MSSQL, but it is not supported by the local version of SqlServer I have installed (2016 SP2), which should still be a supported platform.

**Pull request merged**